docs/setup/README.md

```diff
@@ -0,0 +1,334 @@
+
+If you don't have any state root providers deployed, yet, mosaic can help you deploy anchors on
+origin and auxiliary.
+Anchors are simple, restricted state root providers that you can use to test your deployment.
+
+```js
+const [originAnchor, auxiliaryAnchor] = await Setup.anchors(
+  origin.web3,
+  auxiliary.web3,
+  {
+    // The chain id of the chain that is tracked by this anchor:
+    remoteChainId: auxiliary.chainId,
+    // Anchors use ring buffers to limit the number of state roots they store:
+    maxStateRoots: '10',
+    organization: originOrganization.address,
+    organizationOwner: origin.organization,
+    deployer: origin.deployer,
+  },
+  {
+    // The chain id of the chain that is tracked by this anchor:
+    remoteChainId: origin.chainId,
+    // Anchors use ring buffers to limit the number of state roots they store:
+    maxStateRoots: '10',
+    organization: auxiliaryOrganization.address,
+    organizationOwner: auxiliary.organization,
+    deployer: auxiliary.deployer,
+  },
+  origin.txOptions,
+  auxiliary.txOptions,
+);
+```
+
+## Gateways
+
+You use gateways to transfer your tokens from origin to auxiliary and back.
+Gateways require an organization for management tasks and state root providers in order to enable
+Merkle-Patricia-Proofs for the partner-gateway on the other chain.
+
+Mosaic can help you deploy and activate gateways on origin and auxiliary:
+
+```js
+const [originGateway, auxiliaryCoGateway] = await Setup.gateways(
+  origin.web3,
+  auxiliary.web3,
+  {
+    // The EIP20 token that you want to use for your economy:
+    token: origin.token,
+    // The EIP20 token that is used as the base token on auxiliary:
+    baseToken: origin.baseToken,
+    stateRootProvider: originAnchor.address,
+    // The bounty is the amount that facilitators will earn for progressing stake processes:
+    bounty: '0',
+    organization: originOrganization.address,
+    // The burner address is the address where burnt tokens will be transferred to:
+    burner: origin.burner,
+    deployer: origin.deployer,
+    organizationOwner: origin.organization,
+  },
+  {
+    // The utility token wraps and unwraps base tokens of auxiliary to be transferred as EIP20
+    // tokens through the gateway:
+    utilityToken: auxiliary.utilityToken,
+    stateRootProvider: auxiliaryAnchor.address,
+    // The bounty is the amount that facilitators will earn for progressing stake processes:
+    bounty: '0',
+    organization: auxiliaryOrganization.address,
+    // The burner address is the address where burnt tokens will be transferred to:
+    burner: auxiliary.burner,
+    deployer: auxiliary.deployer,
+    organizationOwner: auxiliary.organization,
+  },
+  origin.txOptions,
+  auxiliary.txOptions,
+);
+```
+
+## Continue Reading
+
+Next, you want to check out how to stake and mint using your new gateways: [Stake and Mint](../stake_and_mint/README.md).
+
+## The entire code
```

Comment:
this should probably be in a js file, and not in the README

Reply:
In an `examples` directory?

Reply:
yes, or in JLP

Reply:
Is JLP our official examples directory? Because I want to link to the file from the docs.

Reply:
Added it to a new `examples` dir